### PR TITLE
Uh Oh Stinky Lo Fi Now Possible Round End Music

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

Per title.

## Why It's Good For The Game

Adds humourous and suitable round-end music everyone will enjoy.

## Changelog
:cl:
add: Uh Oh Stinky But It's Low Fi added as round end music. Credit to LLusion of Soundcloud: https://soundcloud.com/llusionmusic/uh-oh-stinky-but-its-a-soft-lofi-beat
/:cl:

### CODE REVIEW

**Comment by remove32** [comment]
now this is the high quality content we've needed

File: code/game/sound.dm [hunk 1]
```diff
@@ -111,7 +111,7 @@
 /client/proc/playcreditsmusic(vol = 85) // FULPSTATION
 	set waitfor = FALSE
 	if(prefs && (prefs.toggles & SOUND_LOBBY))
-		SEND_SOUND(src, sound('sound/Fulpsounds/Fulp_Piano.ogg', repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC)) // FULPSTATION: Play end music! SO SAD!
+		SEND_SOUND(src, pick(sound('sound/Fulpsounds/Fulp_Piano.ogg', repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC),sound('sound/Fulpsounds/Fulp_Uhoh_Stinky.ogg', repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC))) // FULPSTATION: Play end music! SO SAD!
```
Comment (remove32) [inline code/game/sound.dm:114]: we may at some point want to refactor this into a config item so we can add files and throw em in the round end shuffle

**Comment by remove32** [comment]
verified this works and it is indeed very good

**Comment by Surrealaser** [comment]
@remove32 @TheSwain ready to merge.

**Comment by remove32** [comment]
stinky time

**Pull request merged**